examples/tutorials/nb_python/replay_tutorial.py

```diff
@@ -0,0 +1,440 @@
+# ---
+# jupyter:
+#   accelerator: GPU
+#   colab:
+#     name: Replay Tutorial
+#     provenance: []
+#   jupytext:
+#     cell_metadata_filter: -all
+#     formats: nb_python//py:percent,colabs//ipynb
+#     notebook_metadata_filter: all
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.6.0
+#   kernelspec:
+#     display_name: Python 3
+#     name: python3
+# ---
+
+# %% [markdown]
+# #Gfx Replay Tutorial
+#
+# gfx replay is a feature that lets you save the visual state of the sim, restore the visual state later, and either reproduce earlier observations (from the same camera position) or produce new observations (from different camera positions).
+#
+# Note that restoring the visual state is not the same as restoring the full simulation state; you can't resume simulation from a replay.
```

Comment:
An uninformed user may not understand what all is entailed in "simulation". Can we take the description from the internal post (e.g. "The user can move around in the space (using keyboard control) to view this "object waterfall" from any viewpoint, but she may not interact with these objects.) and add it here as appropriate.